### PR TITLE
Breaking change

### PR DESCRIPTION
**Pull request by pocke**

### CODE REVIEW

File: base.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
-def do_something
+def do_something(p)
   
 end
 
-def do_something2(a, b)
+def do_something2(a, b, c, d = 3)
   
 end
```
Comment (pocke) [inline base.rb]: test

File: base.rb [hunk 2]
```diff
@@ -1,12 +1,12 @@
-def do_something
+def do_something(p)
   
 end
 
-def do_something2(a, b)
+def do_something2(a, b, c, d = 3)
```
Comment (pocke) [inline base.rb:5]: **[method definition validator]**
- Not enough arguments. Did you forget the following arguments? [c] at https://github.com/pocke/sandbox2/blob/rubocop-definition-validator-pr/use.rb#L2

File: base.rb [hunk 3]
```diff
@@ -1,12 +1,12 @@
-def do_something
+def do_something(p)
```
Comment (sumyapp) [inline base.rb]: ああ

File: base.rb (hunk 2 above)
Comment (sumyapp) [inline base.rb:5]: Not enough arguments. Did you forget the following arguments? [c] at   [use.rb#L2](blob/rubocop-definition-validator-pr/use.rb#L2])

Not enough arguments. Did you forget the following arguments? [c] at   [rubocop-definition-validator-pr/use.rb#L2](blob/rubocop-definition-validator-pr/use.rb#L2])

Comment (pocke) [inline base.rb:5]: このリンクの形式いいですね!

**Comment by pocke** [comment]
[use.rb](/pocke/sandbox-2/use.rb#8)

**Comment by pocke** [comment]
https://github.com/pocke/sandbox2/blob/dd1038c314423c66f3606b5286cc21fe00f0c93c/use.rb#L2

**Comment by pocke** [comment]
[use.rb#L2](https://github.com/pocke/sandbox2/blob/dd1038c314423c66f3606b5286cc21fe00f0c93c/use.rb#L2)

**Comment by pocke** [comment]
[use.rb#L2](/blob/rubocop-definition-validator-pr/use.rb#L2])

**Comment by pocke** [comment]
Lint/DefinitionValidator: Not enough arguments. Did you forget the following arguments? [a, b] at [app/controllers/items_controller.rb#L27](/actcat/sideci_rails_sandbox/blob/8f814ddecda9c71f1321383f85407d15ea95f368/app/controllers/items_controller.rb#L27)

**Pull request closed without merging**